src/awaitAll.js

```diff
@@ -0,0 +1,14 @@
+const valueIterator = require('./valueIterator')
+const iterate = require('./iterate')
+const values = require('./values')
+const dump = require('./dump')
+const mapTransformer = require('../transformers/map')
```

Comment:
Actually, I'd like to move *all* methods (internal, transformers) to the `src` folder if that's OK with you